bench-plotter/src/bench_plotter/plotting/dashboard_processor.py

```
@@ -0,0 +1,1193 @@
+            continue
+        start_time = start_ms / 1000
+        end_time = finish_ms / 1000
+
+        try:
+            result = query_with_fallbacks(
+                q=expr,
+                start_time=start_time,
+                end_time=end_time,
+                query_range_func=query_range,
+                instant_query_func=lambda query: asyncio.run(
+                    query_range(query=query, start_unix=start_time, end_unix=end_time)
```

Ruff (0.15.21)

[warning] Function definition does not bind loop variable `start_time`

(B023)